consensus/src/processes/pruning_proof/mod.rs

```diff
@@ -922,18 +944,20 @@ impl PruningProofManager {
                 }
             }
 
+            // Need to ensure this does the same 2M+1 depth that block_at_depth does
             if has_required_block
-                && (root == self.genesis_hash || ghostdag_store.get_blue_score(selected_tip).unwrap() >= required_level_depth)
+                && (root == self.genesis_hash || ghostdag_store.get_blue_score(selected_tip).unwrap() > required_level_depth)
             {
                 break Ok((ghostdag_store, selected_tip, root));
             }
 
             tries += 1;
             if finished_headers {
-                panic!("Failed to find sufficient root for level {level} after {tries} tries. Headers below the current depth of {required_level_0_depth} are already pruned")
+                warn!("Failed to find sufficient root for level {level} after {tries} tries. Headers below the current depth of {required_level_0_depth} are already pruned. Trying anyway.");
+                break Ok((ghostdag_store, selected_tip, root));
             }
             required_level_0_depth <<= 1;
-            warn!("Failed to find sufficient root for level {level} after {tries} tries. Retrying again to find with depth {required_level_0_depth}");
+            trace!("Failed to find sufficient root for level {level} after {tries} tries. Retrying again to find with depth {required_level_0_depth}");
```

Comment:
I think at least for now it should be `warn` since it should be pretty unlikely event that we should be aware of (at least with the safety margin of trying 5/4 of the required depth at first try)